packages/framework/src/client.ts

```diff
@@ -149,6 +150,47 @@ export class Client {
     };
   }
 
+  public getErrors(): string[] {
+    const errors: string[] = [];
+
+    const workflowIds = this.discoveredWorkflows.map((discoveredWorkflow) => discoveredWorkflow.workflowId);
+
+    const workflowOccuranceMap = this.getOccuranceMap(workflowIds);
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏ We can do this check in a single pass with an assert function that will throw an error in the first duplicate detection using a reduce function. The accumulator is the occuranceMap.